scilpy/tractograms/tractogram_operations.py

```diff
@@ -47,6 +68,47 @@ def shuffle_streamlines(sft, rng_seed=None):
     return shuffled_sft
 
 
+def shuffle_streamlines_orientation(sft, rng_seed=None):
+    """
+    Shuffle the orientation of the streamlines. The streamlines are not
+    reversed, only the orientation of the points within the streamlines is
+    shuffled.
```

Comment:
I am not sure I understand the difference.

Reply:
Yes its unclear, it is not a blanket reverse all streamlines, it is a shuffle head/tail of random streamlines